requirements/infrastructure-requirements.md

```diff
@@ -267,7 +270,6 @@ In the case of horizontal scaling (scaling in and out) this requires load-balanc
 
 If the infrastructure is able to directly serve static files, without configuring a web server first,
 this speeds up delivery.
-Moreover, caching these files in a third-party Content Delivery Network (CDN) will reduce latency for users.
```

Comment:
Why did you remove this? I added this originally to illustrate the idea that infrastructure, at it is defined in the introduction, is spread over a number of components (including an external CDN) rather than a single configuration on some server.

Reply:
I think there is a conflict here between this phrase and point 23 of the Software Requirements. Recommending third-party content-delivery networks leads to cross-origin requests and users might be tracked in some form by those networks (like CloudFlare's).

(Ideally I'd rather even recommend limiting cross-origin requests in general, for the same reason)

Reply:
tend to agree with @proycon on CDN's and cross-origin requests.

Maybe it 's good to think/talk about CLARIAH's stance towards using third-party software (which parties can we trust and which do we try to avoid).

If we are clear on this we can still illustrate the requirments with examples using third party software we support.

Also deciding NOT going into this (because it might be too political/frustrating) is good to get consensus about.

Reply:
> Also deciding NOT going into this (because it might be too political/frustrating) is good to get consensus about.

Indeed, it might be best to be a bit neutral on this in the text. Which is what I think we get now we remove this CDN suggestion (and not have a clause discouraging cross-origins like I'd ideally like). Then people can still go either way.

Reply:
Agreed!

Reply:
+1!